### ISSUE TITLE
Remove cOS from GRUB entries

### ISSUE THREAD

**Issue by juadk** [rancher/elemental-toolkit#1568]
We will have to thing about removing cOS from GRUB entries:
![image](https://user-images.githubusercontent.com/6025636/179512465-94e35364-fbd9-4b50-a190-055b617ab319.png)

**Comment by mudler** [comment]
Yes, this issue is specific to elemental. It can also be provided in the elemental config file, which is in yaml
